_posts/2017-07-29-markup-list-of-articles-with-h-feed.md

```diff
@@ -0,0 +1,11 @@
+---
+price: "$10"
+title: "Mark-up the list of articles on ScalaWilliam.com as h-feed"
+skills: [ html, indieweb, semantic, microformats ]
+layout: post
+---
+
+- Issue: [https://github.com/ScalaWilliam/ScalaWilliam.com/issues/64](https://github.com/ScalaWilliam/ScalaWilliam.com/issues/64)
+
+## Scope
```

Comment:
Thanks for pointing this out. Removing from example. https://github.com/ScalaWilliam/Work/commit/875f52be8a9fdcb9e7c8162e175972d24222416d